canonical-web-and-design/cookie-policy
# Translate to Japanese

## Done
- Add Japanese translations

## QA
- Run `npm install`
- Run `npm run build`
- Run `python -m SimpleHTTPServer`
- Visit http://localhost:8000
- Check everything works as before
- Click the link to the Japanese version
- Check everything works and matches [the copy doc](https://docs.google.com/document/d/15KceNeXG9Sm0Lk8_oqoRaIeTCqHzL9_si8VyUeOxI2g/edit#)

Fixes https://github.com/canonical-web-and-design/jp.ubuntu.com/issues/268
